src/pages/overview/Overview.jsx

```diff
@@ -0,0 +1,53 @@
+import {Link} from "react-router-dom";
+import "./Overview.css"
+import axios from "axios";
+import {useEffect, useState} from "react";
+
+function Overview() {
+    const [posts, setPosts] = useState([]);
+    const [loading, setLoading] = useState(true);
+    const [errors, setErrors] = useState(false);
+
+    async function getPosts() {
+        setLoading(true);
+        setErrors(false);
+        try {
+            const result = await axios.get(`http://localhost:3000/posts`);
+            setPosts(result.data);
+            setErrors(false);
+            setLoading(false);
+        } catch (e) {
+            console.error(e);
+            setErrors(true);
+            setLoading(false);
```

Comment:
Je wilt loading sowieso op false zetten of het nou lukt of mislukt, daarom heb je het ook 2x staan. Maar je zou ook finally kunnen toevoegen na de catch waardoor je maar 1x de setLoading op false hoeft te zetten.